ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1313,6 +1313,52 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
+- as: baremetal-compact-agent-ipv4-dhcp-day2-amd-mixarch-f28
+  cluster: build05
+  cron: 1 18 15 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKERS: "1"
+      ADDITIONAL_WORKERS_DAY2: "true"
```

Comment:
Looks like `ADDITIONAL_WORKERS_DAY2: "true"` is the default value. IMO, there is no need to add it there (the same to the other env vars, or else, we will have a long list of env vars, which is hard to maintain).

Reply:
Ok, let me remove it